src/aznfswatchdog

```diff
@@ -65,33 +65,60 @@ chattr -f +i $RANDBYTES
 # kernels. If we find an entry stuck for more than a certain time in SYN_SENT state
 # we delete the entry so that kernel looks up fresh NAT rules and creates a new entry.
 #
-declare -A cthash_synsent
-declare -A cthash_synsent_now
+declare -A cthash_stuck
+declare -A cthash_stuck_now
```

Comment:
for clarity, leave the prev ones and add new one for unreplied state tracking

`#`
`#` Hash for storing how many times we have seen a conntrack entry in UNREPLIED state.
`#` Used for finding if some entry is stuck in UNREPLIED state due to no response from
`#` NFS server or some n/w issue. We want to delete the entry as it prevents creation of
`#` a new conntrack entry, if the IP were to change again soon.
`#`
declare -A cthash_unreplied
declare -A cthash_unreplied_now